### PR TITLE
[T3A5][T13-A1] Aloisius Stephen

### PR DESCRIPTION
**Pull request by aloisiusStephen**
Add Formatter class, an extraction of the responsibility of TextUi class in formatting text for display

### CODE REVIEW

File: src/seedu/addressbook/ui/Formatter.java [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+    public void showGoodbyeMessage() {
+        showToUser(MESSAGE_GOODBYE, DIVIDER, DIVIDER);
+    }
+
+    public void showInitFailedMessage() {
+        showToUser(MESSAGE_INIT_FAILED, DIVIDER, DIVIDER);
+    }
+
+    /** Shows message(s) to the user */
+    public void showToUser(String... message) {
```
Comment (okkhoy) [inline src/seedu/addressbook/ui/Formatter.java]: if the formatter is also doing the printing on screen, it has more than 1 responsibilities now, no? hence effectively violating the SRP.

**Comment by okkhoy** [comment]
@aloisiusStephen why did you close this yourself?

in any case, I have added a few comments, please take a look.

**Comment by aloisiusStephen** [comment]
@okkhoy 
I understood your instruction during the tutorial was to close the PR if i were to re-attempt this activity 
So I did another PR for this Activity [T3A5] under PR #557
I apologize if i misunderstood the instruction

**Comment by okkhoy** [comment]
ah! i did not remember to whom i said that :-P
thanks for clarifying.
-Akshay

**Pull request closed without merging**